docs/content/users/configuration/config_yaml.md

```diff
@@ -1,76 +1,561 @@
+| -- | -- | --
+| :octicons-file-directory-16: project<br>:octicons-globe-16: global | `false` | Can be `true` or `false`; only `true` has any effect.
+
+!!!tip "Workstation configuration required!"
+    See the [NFS section](../install/performance.md#nfs) on the Performance page.
+
+This is typically a global setting. If it’s ever set in both places, the global config will override the project-specific value.
+
+## `ngrok_args`
+
+Extra flags for [configuring ngrok](https://ngrok.com/docs/ngrok-agent/config) when [sharing projects](../topics/sharing.md) with the `ddev share` command.
+
+| Type | Default | Usage
+| -- | -- | --
+| :octicons-file-directory-16: project | `` |
+
+Example: `--basic-auth username:pass1234`.
+
+## `no_bind_mounts`
+
+Whether to not use Docker bind mounts.
+
+| Type | Default | Usage
+| -- | -- | --
+| :octicons-globe-16: global | `false` | Can `true` or `false`.
+
+Some Docker environments (like remote Docker) do not allow bind mounts, so when `true` this turns those off, turns on Mutagen, and uses volume copies to do what bind mounts would otherwise do.
+
+## `no_project_mount`
+
+Whether to skip mounting project into web container.
+
+| Type | Default | Usage
+| -- | -- | --
+| :octicons-file-directory-16: project | `false` | Can be `true` or `false`.
+
+!!!warning "Advanced users only!"
+    When `true`, project will not be mounted by DDEV into the web container. Enables experimentation with alternate file mounting strategies.
+
+## `nodejs_version`
+
+Node.js version for the web container’s “system” version.
+
+| Type | Default | Usage
+| -- | -- | --
+| :octicons-file-directory-16: project | current LTS version | Can be `14`, `16`, or `18`.
+
+`nvm` is also available inside the container and via `ddev nvm`, and can be set to any valid version including much older ones.
+
+## `omit_containers`
+
+Containers that should not be loaded automatically for one or more projects.
+
+| Type | Default | Usage
+| -- | -- | --
+| :octicons-file-directory-16: project<br>:octicons-globe-16: global | `[]` | **For projects**, can include `db`, `dba`, and `ddev-ssh-agent`.<br>**Globally**, can include `dba` and `ddev-ssh-agent` only.
+
+Example: `omit_containers: [db, dba, ddev-ssh-agent]` starts the project without a `db` container, phpMyAdmin, and SSH agent. Some containers can be omitted globally in `~/.ddev/global_config.yaml` and the result is additive; all containers named in both places will be omitted.
+
+!!!warning
+    Omitting the `db` container will cause database-dependent DDEV features to be unstable.
+
+## `override_config`
+
+Whether to override config values instead of merging.
+
+| Type | Default | Usage
+| -- | -- | --
+| :octicons-file-directory-16: project | `false` | Can be `true` or `false`.
+
+When `true`, the `config.*.yaml` file with the option will have its settings *override* rather than *merge with* others. Allows statements like `nfs_mount_enabled: false` or `additional_hostnames: []` to work.
```

Comment:
Please link to https://ddev.readthedocs.io/en/latest/users/extend/customization-extendibility/#extending-configyaml-with-custom-configyaml-files

Reply:
[Done](https://github.com/drud/ddev/pull/4332/commits/2b85a55ad64b9339f194cbd6a9e1bcb9d80a3634).